src/workerd/server/workerd.capnp

```diff
@@ -601,6 +601,14 @@ struct Worker {
     # workerd uses SQLite to back all Durable Objects, but the SQL API is hidden by default to
     # emulate behavior of traditional DO namespaces on Cloudflare that aren't SQLite-backed. This
     # flag should be enabled when testing code that will run on a SQLite-backed namespace.
+
+    container @5 :ContainerOptions;
+    # Optional configuration options for local development container interaction.
+
+    struct ContainerOptions {
+      name @0 :Text;
+      # Name of the container associated with the current DO namespace.
```

Comment:
How dose this work? Each DO instance gets its own container. Is this the name of the container _image_? This really needs a lot more commentary.